integration-tests/volumes/partner/responses.yml

```diff
@@ -84,4 +84,17 @@ tablet:
     ios:
       status_code: 200
       headers: []
-      content: "hello world"
```

Comment:
This response content is used for verifying that Contile correctly handles invalid tiles responses from the partner API in `error_tablet_ios_reqwest_error`. It is not supposed to match the schema of an actual response to a mobile client.

Reply:
We needed a reliable way to return invalid tiles responses from the partner API in integration tests. While in prod iPads are expected to receive valid tiles data, we programmed the partner service in the integration tests to send an invalid response content for requests for form-factor `tablet` and os-family `ios`.

We control the User-Agent header in the `integration-tests/volumes/client/scenarios.yml` and know that Contile will receive an invalid response content from the partner and can verify that Contile handles that as expected.